src/SourceBuild/content/repo-projects/Directory.Build.props

```
@@ -142,7 +142,7 @@
     <EnvironmentVariables Include="DeterministicSourcePaths=false" Condition="'$(DeterministicBuildOptOut)' == 'true'" />
 
     <EnvironmentVariables Include="SourceRoot=$(ProjectDirectory)" />
-    <EnvironmentVariables Include="DotNetBuildFromSourceFlavor=Product" />
+    <EnvironmentVariables Include="DotNetBuildFromSourceFlavor=Product" Condition="'$(DotNetBuildSourceOnly)' == 'true'" />
```

```suggestion
    <EnvironmentVariables Include="DotNetBuildFromSourceFlavor=Product" />
```

Removing this change for now. There are a couple places where we use Flavor in runtime still.